litmus-portal/backend/graphql-server/graph/schema.graphqls

```diff
@@ -33,14 +33,22 @@ type ClusterEvent {
   cluster: Cluster!
 }
 
+type ActionPayload {
```

Comment:
This is required in case of the subscriber as well as the workflow-agent, thus having k8s_manifest, namespace etc. compulsory might be a problem. We can have all this inside the execution_data itself which can then be marshalled to appropriate struct depending on the request_type.

Reply:
Yes, these should not be compulsory. I have removed "!" from ActionPayload